{"uuid": "8fd3cfbc-d313-4aa7-b51d-25afb41497eb", "children": ["6ef5b572-7588-4313-8d5e-55affe9f254e", "f35f798c-0f02-488f-84b0-c1b8ddbbf988", "16e28f40-5498-42f5-aabe-6cde0eadd0a3"], "befores": [{"name": "_UnitTestCase__pytest_class_setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: chrome not reachable\n  (Session info: chrome=83.0.4103.116)\n  (Driver info: chromedriver=2.37.544315 (730aa6a5fdba159ac9f4c1e8cbc59bf1b5ce12b7),platform=Windows NT 10.0.18362 x86_64)\n\n", "trace": "  File \"c:\\python38\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\python38\\lib\\site-packages\\_pytest\\fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\python38\\lib\\site-packages\\_pytest\\fixtures.py\", line 788, in call_fixture_func\n    res = next(it)\n  File \"c:\\python38\\lib\\site-packages\\_pytest\\unittest.py\", line 99, in fixture\n    setup()\n  File \"C:\\Users\\admin\\PycharmProjects\\JKDDSystem\\src\\test_SWJK.py\", line 16, in setUpClass\n  File \"E:\\LocalYHYH\\eles\\eles.py\", line 90, in system_login\n    self.system_input_username(\"dym6\")\n  File \"E:\\LocalYHYH\\eles\\eles.py\", line 74, in system_input_username\n    self.eles_input(operations.public_username_input,username)\n  File \"E:\\LocalYHYH\\eles\\eles.py\", line 40, in eles_input\n    ele = self.eles_find_element(loc)\n  File \"E:\\LocalYHYH\\eles\\eles.py\", line 17, in eles_find_element\n    return WebDriverWait(self.driver, timeout = timeout, poll_frequency = poll_frequency).until(lambda x:x.find_element(*loc))\n  File \"c:\\python38\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 71, in until\n    value = method(self._driver)\n  File \"E:\\LocalYHYH\\eles\\eles.py\", line 17, in <lambda>\n    return WebDriverWait(self.driver, timeout = timeout, poll_frequency = poll_frequency).until(lambda x:x.find_element(*loc))\n  File \"c:\\python38\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 976, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\n  File \"c:\\python38\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\python38\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1594377252667, "stop": 1594377260685}], "start": 1594377252667, "stop": 1594377260827}